config/peerpods/podvm/libvirt-podvm-image-handler.sh

```diff
@@ -158,6 +163,183 @@ function download_rhel_kvm_guest_qcow2() {
 
 }
 
+# Function to extract RVPS value from PODVM SE image
+function calculate_rvps_qcow2() {
+    mkdir -p "${CAA_SRC_DIR}"/podvm/rvps_config
+    cp "${CAA_SRC_DIR}"/podvm/output/*.qcow2 "${CAA_SRC_DIR}"/podvm/rvps_config/podvm-libvirt.qcow2
```

Comment:
with this copy, we will have 2 copies of the qcow2 file along with it's original one.
```
"${CAA_SRC_DIR}"/podvm/output/*.qcow2 
/payload/podvm-libvirt.qcow2
"${CAA_SRC_DIR}"/podvm/rvps_config/podvm-libvirt.qcow2
``` 
if we are not doing any destructive actions, how about using one of the qcow2 image from the above 2?

Reply:
I had tried it. But deleting one of those was creating issue somehow so I am keeping both.